docs/deployments/open5gs-with-bgp/values/eupf.yaml

```diff
@@ -115,3 +120,22 @@ readinessProbe:
     port: http
   initialDelaySeconds: 5
   periodSeconds: 10
+
+ui:
```

Comment:
please define default settings for ui app in file `.deploy/helm/eupf/values.yaml`

Reply:
Done